src/cmd/repository.go

```diff
@@ -56,5 +62,6 @@ var listRepositoryCmd = &cobra.Command{
 
 func init() {
 	getCmd.AddCommand(getRepositoryCmd)
+	listRepositoryCmd.Flags().Bool("hidden", false, "list only hidden repositories when set")
```

Comment:
what about `show-hidden` to be a little more directly readable `opslevel list repos --show-hiden`

Reply:
oh i see it inverts the list 🤔 is that useful - should it just add them to the list?

Reply:
if it inverts then it should be `--only-hidden`

Reply:
Good suggestion 👍 - I'll update this